Makefile.am

```
@@ -196,6 +196,17 @@ musig_example_LDFLAGS += -lbcrypt
 endif
 TESTS += musig_example
 endif
+if ENABLE_MODULE_FROST
+noinst_PROGRAMS += frost_example
+frost_example_SOURCES = examples/frost.c
+frost_example_CPPFLAGS = -I$(top_srcdir)/include
```

```suggestion
frost_example_CPPFLAGS = -I$(top_srcdir)/include -DSECP256K1_STATIC
```

This should fix the CI failures on mingw.

---

Thanks! 🙏